incuna-sass/functions/_functions-variables.sass

```diff
@@ -0,0 +1,5 @@
+// Set a default font-base variable here that can
+// be overwritten locally
+// This way we can import incuna-sass files above
+// locally defined variables without getting errors
+$font-base: 16px !default
```

Comment:
I don't think we need to place this file in the `functions/` folder. The current contents aren't particular to the Sass within that folder, even if that's currently the only place that uses it. Instead i think this should be `incuna-sass/_variables.sass` file.